src/WC_Gateway_ThawaniGateway.php

```diff
@@ -269,7 +274,14 @@ public function init_form_fields()
             //     'description' => '',
             //     'default' => 'no',
             // ),
-            'environment' => array(
+            'status' => array(
+                'title' => __('Select complete order status', 'thawani'),
```

Comment:
@w7shdev what do you think about adding a little tooltip for users to explain this a bit more. I think people that are not technical will not get this immediately. Something like:

```php
                'description' => __('This is the status that your order will be in after payment is completed', 'thawani'),
                'desc_tip' => true,
```

Reply:
Sure, totally missed this point